web-marketplace/src/pages/Dashboard/hooks/useProfileItems.tsx

```diff
@@ -0,0 +1,23 @@
+import { useQueryIfCreditClassCreator } from 'hooks/useQueryIfCreditClassCreator';
+import { useQueryIsClassAdmin } from 'hooks/useQueryIsClassAdmin';
+import { useQueryIsIssuer } from 'hooks/useQueryIsIssuer';
+import { useQueryIsProjectAdmin } from 'hooks/useQueryIsProjectAdmin';
+
+export const useProfileItems = () => {
+  const isIssuer = useQueryIsIssuer();
+  const isCreditClassCreator = useQueryIfCreditClassCreator();
+  const isProjectAdmin = useQueryIsProjectAdmin();
+  const isCreditClassAdmin = useQueryIsClassAdmin();
+
+  const showProjects = isIssuer || isProjectAdmin;
+  const showCreditClasses = isCreditClassCreator || isCreditClassAdmin;
+
+  return {
+    showProjects,
+    // Explictly set to false until we enable the class creation flow
```

Comment:
disabling credit classes dropdown item/tab for now since the class creation flow isn't done and we don't have any implementation yet for displaying credit classes cards created by the user